[C# Analyzer] Configuration Versioning Note (#81)
mongodb/docs-visual-studio-extension: source/configuration.txt

         its nested namespaces only
       - ``"None"``: Do not analyze LINQ expressions
       
       .. important::

>           The ``LinqAnalysisVerbosity`` configuration option is only available for {+product-short+} v2.0 and above.

Reviewer: S: style guide - earlier/later - https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#e:~:text=E-,earlier,-later%20Use%20these + placement of `only`
```suggestion
          The ``LinqAnalysisVerbosity`` configuration option is available only in {+product-short+} v2.0 and later.
```
